docs/sources/setup-grafana/configure-security/configure-scim-provisioning/configure-scim-with-azuread/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Learn how to configure SCIM provisioning with Azure AD in Grafana Enterprise. This guide provides step-by-step instructions for setting up automated user and team management, including enterprise application configuration, service account creation, attribute mapping, and provisioning settings to ensure seamless integration between Azure AD and Grafana.
+keywords:
+  - grafana
+  - scim
+  - azure
+  - azure ad
+  - entra id
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Configure SCIM with Azure AD
+title: Configure SCIM with Azure AD
+weight: 320
+---
+
+# Configure SCIM with Azure AD
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{< /admonition >}}
+
+This guide explains how to configure SCIM provisioning with Azure AD to automate user and team management in Grafana.
+
+{{< admonition type="note" >}}
+This feature is behind the `enableSCIM` feature toggle.
+You can enable feature toggles through configuration file or environment variables.
+
+For more information, refer to the [feature toggles documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
+{{< /admonition >}}
+
+## Prerequisites
+
+Before configuring SCIM with Azure AD, ensure you have:
+
+- Grafana Enterprise or Grafana Cloud Advanced
+- Admin access to both Grafana and Azure AD
+- SAML authentication configured between Grafana and Azure AD
+- SCIM feature enabled in Grafana
+
+## Configure SCIM in Grafana
+
+To enable SCIM provisioning in Grafana, create a service account and generate an access token that will be used to authenticate SCIM requests from Azure AD.
+
+### Create a service account
+
+1. Navigate to **Administration > User Access > Service accounts**
```

Comment:
I think this part should contain info about the required basic/fixed roles for the SA.